packages/main/src/Tokenizer.js

```diff
@@ -35,6 +35,9 @@ const metadata = {
 		 */
 		expanded: { type: Boolean },
 		_nMoreText: { type: String },
+		_hideNMoreText: {
```

Comment:
I think the `showMore` property is for the same purpose, do we need another one?